.travis.yml

```diff
@@ -15,6 +26,10 @@ before_install:
 script:
 - ./mvnw clean install
 
+after_success:
+- 'if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then openssl aes-256-cbc -K $encrypted_9047eb4b7969_key -iv $encrypted_9047eb4b7969_iv -in cd/codesigning.asc.enc -out cd/codesigning.asc -d; gpg --fast-import cd/codesigning.asc; fi'
+- 'if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then mvn deploy -Prelease --settings cd/settings.xml; fi'
```

Comment:
isn't it strange to use release profile when we are not releasing real version?

Reply:
Actually, we are releasing but the snapshot. The mechanism is same for releasing to (SNAPSHOT/Release) - only target repository is different.